src/components/Hidayah/index.js

```diff
@@ -17,12 +19,50 @@ const PopUpModal = ({ popUpClassName, displayPopUpModal, setDisplayPopUpModal, p
   </div>);
 };
 
+const Levels = ({ level, setLevel }) => (<div>
+  { LEVELS && LEVELS.map((lvl) => (
+    <>
+      <input type="radio" id={lvl} name={lvl} value={lvl} checked={lvl === level} onChange={() => setLevel(lvl)} />
+      <label key={lvl} for={lvl}>{lvl}</label>
+    </>
+  ))}
+</div>);
+
+const SettingsModal = ({ settings, setSettings, hideModal, settingsModalClassName }) => (<div style={{ position: 'absolute', top: 0, left: 0, width: '100%', height: '100%', padding: '1%', backgroundColor: '#ddd' }} className={settingsModalClassName || ''}>
+  <button style={{ position: 'absolute', top: '1%', right: '1%', zIndex: 1 }} onClick={hideModal}>X</button>
+  <h3>Settings</h3>
+  <h4>Levels:</h4>
```

Comment:
How will you handle Arabic?

Reply:
I think this text should be something that a client can set. So allow a title component

Reply:
> How will you handle Arabic?

Perhaps we can have a file contains all the static text ( since we are going to export many languages in a folder struc way /ar/modal name /en/modal name )

Reply:
We can then also allow the user to edit those by sending an object to override the needed words, but let's keep it in a seperated PR